Routing/Loader/Reader/RestActionReader.php

```diff
@@ -0,0 +1,631 @@
+<?php
+
+/*
+ * This file is part of Handcrafted in the Alps - Rest Routing Bundle Project.
+ *
+ * (c) Sulu GmbH <hello@sulu.io>
```

Comment:
Does MIT allow to just change the copyright notice?

Reply:
Oh thats a good question didn't thought about that 🙈 Does somebody know?
Else if its ok for you I could just let the FOS header.

Reply:
As I read the copy holders need to stay and multiple can be added.